specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2025-08-01-preview/examples/RedisEnterpriseMigrationStart.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "clusterName": "cache1",
+    "resourceGroupName": "rg1",
+    "api-version": "2025-08-01-preview",
+    "subscriptionId": "e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f",
+    "parameters": {
+      "sourceResourceId": "/subscriptions/e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f/resourceGroups/rg1/providers/Microsoft.Cache/redis/cache1",
+      "switchDns": true,
+      "skipDataMigration": true
+    }
+  },
+  "responses": {
+    "200": {},
+    "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=5&length=3)

resource creates should normally use 201+provisioning state instead of 202 accepted